szachy.css

```diff
@@ -32,10 +32,17 @@ body {
 	margin-top: 2em;
 	margin-left: auto;
 	margin-right: auto;
+	background: blue;
 	flex-wrap: wrap;
 	width: 600px;
 	height: 600px;
-	border: 5px #444 solid;
+	border: 35px #444 solid;
```

Comment:
Way of implementing it (with border-image) is ok, but could be done with wrapper-div to achive more flexibility

Reply:
Atleast try it that way to understand other ways to achive it